doc/api/fs.md

```diff
@@ -602,7 +602,9 @@ default value of 64 kb for the same parameter.
 
 `options` can include `start` and `end` values to read a range of bytes from
 the file instead of the entire file.  Both `start` and `end` are inclusive and
-start at 0. The `encoding` can be any one of those accepted by [`Buffer`][].
+start counting at 0. If `fd` is specified and `start` is omitted or undefined,
```

Comment:
Nit: Could you put `undefined` in backticks so it’s clear you are referring to the JS value?